ui/components-react/src/components-react/filter-builder/FilterBuilderToolbar.tsx

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from "react";
+import cx from "classnames";
+import { Flex, type FlexProps, IconButton } from "@itwin/itwinui-react";
+import { SvgAdd, SvgDelete } from "@itwin/itwinui-icons-react";
+import { UiComponents } from "../UiComponents";
+
+/**
+ * Props for [[PropertyFilterBuilderToolbar]] component.
+ * @internal
+ */
+interface PropertyFilterBuilderToolbarProps {
+  /** Size to render the component. If undefined, defaults to iTwinUI "medium" size. */
+  size?: "small" | "large";
+  /** Function to add child of current group. */
+  onAddChild?: () => void;
+  /** Function to remove child from current group. */
+  onDelete?: () => void;
```

Comment:
I think these should be made required as buttons for these actions are always rendered.